api/api/models/vulns.py

```diff
@@ -60,9 +61,9 @@ class Meta:
     creation_date: models.DateTimeField = models.DateTimeField(auto_now_add=True, editable=False)
     last_updated_date: models.DateTimeField = models.DateTimeField(auto_now_add=True, editable=True)
 
-    author: models.ForeignKey = models.ForeignKey(
+    author: Optional[Auth] = models.ForeignKey(
         Auth, on_delete=models.CASCADE, related_name='author', blank=True, null=True)
-    last_editor = models.ForeignKey(Auth, on_delete=models.CASCADE, related_name='last_editor')
+    last_editor: Auth = models.ForeignKey(Auth, on_delete=models.CASCADE, related_name='last_editor')
 
-    vuln_type = models.OneToOneField(VulnType, on_delete=models.CASCADE, blank=True)
-    images = models.ManyToManyField(ImageModel, blank=True, default=None)
+    vuln_type: VulnType = models.OneToOneField(VulnType, on_delete=models.CASCADE, blank=True)
+    images: List[ImageModel] = models.ManyToManyField(ImageModel, blank=True, default=None)
```

Comment:
You don't have to correct that, but adding type at the initialization of a variable is a bit useless. We shall add it when the compiler will have to figure out during runtime. That's not the case here

Reply:
this is just to make the linter happier, not for initializing data for the interpreter, since python doesn't care about it